src/it/projects/mdep-602-log-on-error-level-when-failOnWarning/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.maven.plugins.dependency</groupId>
+  <artifactId>mdep-602-log-on-error-level-when-failOnWarning</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+
+  <description>Test that dependency:analyze logs dependency warnings as errors when failOnWarning is set to true</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-project</artifactId>
+      <version>2.0.6</version>
```

Comment:
please use something newer - 3.2.5

Reply:
I copy-pasted this from another test. :grimacing: I can bump the version, but 3.2.5 doesn't seem available on Maven Central. :eyes: https://search.maven.org/artifact/org.apache.maven/maven-project

Reply:
ok, leave as is. 

It will be different task to upgrade Maven version in tests 😄